xmtp_api_grpc/src/grpc_api_helper.rs

```diff
@@ -32,8 +32,11 @@ use xmtp_proto::{
 async fn create_tls_channel(address: String) -> Result<Channel, Error> {
     let channel = Channel::from_shared(address)
         .map_err(|e| Error::new(ErrorKind::SetupError).with(e))?
+        .initial_connection_window_size(Some((1 << 31) - 1))
         .keep_alive_while_idle(true)
```

Comment:
**Purpose:** Configures whether the client should send keep-alive pings to the server when the connection is idle.

**Functionality:** When set to true, this option ensures that periodic pings are sent on an idle connection to keep it alive and detect if the server is still responsive.

**Impact:** This helps maintain active connections, particularly through NATs, load balancers, and other middleboxes that might drop idle connections. It helps ensure that the connection is promptly usable when new requests need to be sent.

Reply:
Should we drop these descriptions in as code comments? They are going to be very helpful for future maintenance